applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -1306,12 +1317,58 @@ int unicast_client_stop(void)
 	return 0;
 }
 
+static bool ts_valid(enum timestamp_state ts_state)
+{
+	return (ts_state == TS_VALID);
+}
+
+#define SDU_INTERVAL_US (10000u)
+/**
+ * @brief
+ *
+ * @param tx_info
+ * @param ts_state
+ * @return uint32_t
+ */
+uint32_t timestamp_select(struct bt_iso_tx_info const *const tx_info)
+{
+
+	if (!ts_valid(ts_state[AUDIO_CH_L]) && !ts_valid(ts_state[AUDIO_CH_R])) {
```

Comment:
The two following `if` statements could be `else if` to show that the states we are checking excludes each other.